gpt_playground/scripts/shell_helpers.py

```diff
@@ -0,0 +1,37 @@
+from apps.assistants.models import OpenAiAssistant
+from apps.experiments.models import Experiment, ExperimentSession
```

Comment:
these will need to be moved to inline imports since this script will run before Django has run it's setup code